src/components/queryBuilder/views/LogsQueryBuilder.tsx

```diff
@@ -0,0 +1,310 @@
+import React, { useEffect, useMemo, useRef } from 'react';
+import { ColumnsEditor } from '../ColumnsEditor';
+import { Filter, OrderBy, QueryBuilderOptions, SelectedColumn, ColumnHint, DateFilterWithoutValue, FilterOperator, OrderByDirection, TableColumn } from 'types/queryBuilder';
+import { ColumnSelect } from '../ColumnSelect';
+import { OtelVersionSelect } from '../OtelVersionSelect';
+import { OrderByEditor, getOrderByOptions } from '../OrderByEditor';
+import { LimitEditor } from '../LimitEditor';
+import { FiltersEditor } from '../FilterEditor';
+import allLabels from 'labels';
+import { getColumnByHint } from 'data/sqlGenerator';
+import { columnFilterDateTime, columnFilterString } from 'data/columnFilters';
+import { Datasource } from 'data/CHDatasource';
+import { useBuilderOptionChanges } from 'hooks/useBuilderOptionChanges';
+import { versions as otelVersions } from 'otel';
+import { Alert, VerticalGroup } from '@grafana/ui';
+import useColumns from 'hooks/useColumns';
```

Comment:
Why are  these separate imports from the hooks below?

Reply:
Depends on what you mean, I think the imports might have gotten out of order at some point. They're also in different files, although we could group them into one file, especially since some of them aren't default exports